datalad/support/sshconnector.py

```diff
@@ -118,6 +119,28 @@ def close(self):
             else:
                 raise
 
+    def copy(self, source, destination):
+        """Copies source file/folder to destination on the remote.
+
+        Parameters
+        ----------
+        source: str
+          file/folder path to copy from on local
```

Comment:
Would be nice if it supported list of source paths as well ;-) pretty much how regular scp does